### PR TITLE
open-cluster-management/backlog#3662 ts res exist

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Review by nathanweatherly — APPROVED** [review]

File: install/uninstall.adoc [hunk 1]
```diff
@@ -13,6 +13,9 @@ This adds the removal of other required components and settings to the items tha
 When you continue with this step, it removes all of the components and subscriptions that were not removed with the custom resource removal.
 If you complete this level of unistallation, you must reinstall the operator before reinstalling the custom resource.
 
++
+IMPORTANT: Before unistalling the Red Hat Advanced Cluster Management hub cluster, you must detach all of the clusters that are managed by that hub cluster. For more information about detaching clusters, see the *Removing a cluster from management* section by selecting the information for your provider in  link:../manage_cluster/create.adoc#creating-a-cluster-with-red-hat-advanced-cluster-management-for-kubernetes[Creating a cluster with Red Hat Advanced Cluster Management for Kubernetes].
```
Comment (dockerymick) [inline install/uninstall.adoc]: I suggest that you bold IMPORTANT

Also update the sentence to: Before you uninstall the...

I think you should remove the anchor, but honestly it's best to keep it for the sake of the build lol

File: troubleshoot_acm/trouble_uninstall_detach [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[#troubleshooting-failed-unistallation-because-resources-exist]
+= Troubleshooting failed uninstallation because resources exist
+
+[#symptom-failed-unistallation-because-resources-exist]
+== Symptom: Uninstallation fails because resources exist.
+
+When unistalling Red Hat Advanced Cluster Management for Kubernetes, the installation fails with the following error message:
```
Comment (dockerymick) [inline troubleshoot_acm/trouble_uninstall_detach]: Update to: When you uninstall...

File: troubleshoot_acm/troubleshooting.adoc [hunk 3]
```diff
@@ -62,8 +62,9 @@ View the list of troubleshooting topics for Red Hat Advanced Cluster Management
 
 To get to the original installing tasks, view link:../install/install_overview.adoc[Installing].
 
-* xref:../troubleshoot_acm/trouble_reinstall.adoc#troubleshooting-reinstallation-failure[Troubleshooting reinstallation failure]
+ ** xref:../troubleshoot_acm/trouble_reinstall.adoc#troubleshooting-reinstallation-failure[Troubleshooting reinstallation failure]
 
+ ** xref:../troubleshoot_acm/trouble_unistall_detach.adoc##troubleshooting-failed-unistallation-because-resources-exist[Troubleshooting failed uninstallation because resources exist]
```
Comment (dockerymick) [inline troubleshoot_acm/troubleshooting.adoc]: remove one hashtag from the anchor

**Review by dockerymick — APPROVED** [review]
Made a few comments, looks good Chris. Thanks

File: install/uninstall.adoc (hunk 1 above)
Comment (cadawson) [reply install/uninstall.adoc]: Isn't Important automatically bold?  I thought the build makes it bold. I guess I will find out.

Changed "Before..."

Kept the anchor. Now you, too? ;-)

File: troubleshoot_acm/trouble_uninstall_detach (hunk 2 above)
Comment (cadawson) [reply troubleshoot_acm/trouble_uninstall_detach]: Done

File: troubleshoot_acm/troubleshooting.adoc (hunk 3 above)
Comment (cadawson) [reply troubleshoot_acm/troubleshooting.adoc]: Good catch there. One less link I gotta worry about.

**Pull request merged**